packages/react/babel.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Babel preset to provide React support for Nx.
+ */
+
+module.exports = function (api: any, options: {}) {
```

Comment:
We're only using react preset for now, but we can check what CRA and Gatsby is doing and improve this in the future.